lib/sortChannels.ts

```diff
@@ -1,13 +1,55 @@
-import { ChannelStats } from "@/types/ChannelStats";
+import { ChannelStats, RankCriterion } from "@/types/ChannelStats";
 
 function sortChannels(channelStats: ChannelStats[]) {
-  return channelStats.sort((a, b) => {
-    return (
-      (b.numberOfNotes! - a.numberOfNotes!) ||
-      (b.numberOfSongs - a.numberOfSongs) ||
-      (b.numberOfCurators - a.numberOfCurators)
-    );
+  const criteria: RankCriterion[] = [
+    { name: 'numberOfNotes', weight: .6 },
+    { name: 'numberOfSongs', weight: .2 },
+    { name: 'numberOfCurators', weight: .2 },
```

Comment:
would you mind changing the weightings to equal weightings of `0.333` per criteria?
- this was a change requested yesterday, sorry for the last minute changes